scripts/migrate.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	_ "github.com/go-sql-driver/mysql"
+
+	"github.com/cycloidio/terracost/mysql"
+)
+
+func main() {
+	db, err := sql.Open("mysql", "root:terracost@tcp(localhost:33060)/terracost_test?multiStatements=true")
+	if err != nil {
+		panic(err)
```

Comment:
Change it for a `log.Fatalf` I would say.

Same for the next one.

Reply:
Changed